### ISSUE TITLE
Error: Duplicate "graphql" modules cannot be used at the same time

### ISSUE THREAD

**Issue by hagiosofori** [neo4j-graphql/neo4j-graphql-js#57]
Currently encountering the problem above.

tried the following:

- using `yarn --flat`
- editing graphql version in package.json to be the same version as found in neo4j-graphql-js/package.json
- changing graphql version in neo4j-graphql-js/package.json to be the same as found in my own package.json.

none of these has worked so far.

any help please? Is there something I am missing?

**Comment by DveMac** [comment]
This is because package.json specifies `"graphql": "git://github.com/graphql/graphql-js.git#npm",` - which is now an rc version.

I forked and built my own version to fix/hack the issue

**Comment by jexp** [comment]
There was an issue with different graphql modules in grandstack-starter, that's why I added the resolution with an RC.

See here: 
https://github.com/grand-stack/grand-stack-starter/blob/master/api/package.json#L26-L28

**Comment by jexp** [comment]
I think we should be able to do it here too, but I want to wait for @johnymontana to return do to the update & release.

**Comment by poteto** [comment]
I had the same issue, ensuring only one version `graphql` was installed solved it:

```
// package.json
"resolutions": {
  "neo4j-graphql-js/graphql": "<your graphql version here>"
}
```

**Comment by jexp** [comment]
ya we tied it to: `v14.0.0-rc.2`

**Comment by johnymontana** [comment]
This should be solved by b8a0a7f6a7698ac4c83690ed8950bca892d11c93

Previously we were relying on a release candidate version of graphql-js to be able to use the extend type functionality for schema augmentation. However, we've refactored the way we augment the schema and have moved back to using the latest stable release of graphql-js.
